styles.css

```
@@ -33,8 +33,22 @@ body {
 .desktop-browse-layout {
     padding-top: 10px;
 }
+
 .desktop-metrics-layout {
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)